crates/starknet-server/tests/test_advancing_time.rs

```diff
@@ -72,22 +67,177 @@ mod advancing_time_tests {
             .await
             .unwrap();
 
-        let new_contract_address = get_udc_deployed_address(
+        get_udc_deployed_address(
             FieldElement::ZERO,
             declaration_result.class_hash,
             &starknet_rs_core::utils::UdcUniqueness::NotUnique,
             &[],
+        )
+    }
+
+    pub async fn get_current_timestamp(
+        devnet: &BackgroundDevnet,
+        contract_address: FieldElement,
+    ) -> FieldElement {
+        let call_current_timestamp = FunctionCall {
+            contract_address,
+            entry_point_selector: get_selector_from_name("get_timestamp").unwrap(),
+            calldata: vec![],
+        };
+        devnet
+            .json_rpc_client
+            .call(call_current_timestamp.clone(), BlockId::Tag(BlockTag::Latest))
+            .await
+            .unwrap()[0]
+    }
+
+    #[tokio::test]
+    async fn timestamp_syscall_set_in_past() {
+        let devnet: BackgroundDevnet =
+            BackgroundDevnet::spawn().await.expect("Could not start Devnet");
+        let new_contract_address = setup_timestamp_contract(&devnet).await;
```

Comment:
Here and in other places, it would really be more indicative if the variable was not called `new_contract_address` but `timestamp_contract_address`.

Reply:
true, renamed in 5b8dba1ad980cd3310fd4a14ed78eb636e12b8ea